docker/Release.plan

```diff
@@ -1,9 +1,15 @@
-FROM progrium/busybox
+FROM alpine:3.3
 MAINTAINER Philip Cunningham <hello@filib.io>
 
-ENV LANG en_US.UTF-8
+ADD docker/files/ .
```

Comment:
Whoops; forgot to commit the `.apk` files being added here. Will do in a moment.